### DISCUSSION TITLE
Where to put checkpoints?

### DISCUSSION THREAD

**Discussion by Creepybits** [IndigoDosSantos/stable-cascade-one-click-installer#33] [Q&A] [answered]
I have the cascade checkpoints downloaded and have been using it through another webui up until it stopped working a few days ago. I've followed the stepps and installed all requirements etc and the webui starts, but everything that comes out of it is way below any standard I've seen for cascade. 

I've been searching my local directories for where I can place the checkpoints I've downloaded, but I can't seem to find it anywhere. Should I just create a folder for the checkpoints or am I missing something else?

**Comment by IndigoDosSantos** [comment]
>"everything that comes out of it is way below any standard I've seen for cascade."

Hmm, really strange, 🤔 one-click installer uses bfloat16 Cascade models. These should provide similar quality to float32 models. If the quality is significantly worse, something else might be wrong.

* **Model Location:** By default, the diffusers library (which one-click-installer uses) stores models in a hidden folder within the Hugging Face hub. Check this location: 
   `"C:\Users\Username\.cache\huggingface\hub"` 
   If you specified a different location, navigate there instead.

**Reply by Creepybits** [reply]
Yeah seems I have some other issues I can't seem to fix.

![Skärmbild 2024-03-20 204244](https://github.com/EtienneDosSantos/stable-cascade-one-click-installer/assets/60452756/2d15e1dc-fda0-443e-9585-c1024ef28dc2)

No matter what I installe it comlains that I have the wrong version.

And also get an error message about issues with:

\AppData\Local\Programs\Python310\Lib\site-packagees\xformers\_C.pyd
![image](https://github.com/EtienneDosSantos/stable-cascade-one-click-installer/assets/60452756/a7c7028d-456b-4378-83d4-de2368a004d7)

And I simply don't know enough to even begin to know which questions to ask

**Reply by IndigoDosSantos** [reply, marked as answer]
Aha! Seeing `xformers` is a big clue. The one-click installer doesn't use that, which usually means the virtual environment isn't set up right.

Can you open your "stable-cascade-one-click-installer" folder and look for a folder called "venv"? If it's missing, let's try a fresh install:

1. Delete the entire "stable-cascade-one-click-installer" folder.
2. Re-run the installer.
3. Double-check that you now have a "venv" folder inside.

**Reply by Creepybits** [reply]
Yeah there's no venv. I'll try a re-installment and see what happens. Thanks!

**Reply by Creepybits** [reply]
Now it works! Thank you! I have no idea why the first install got botched. 

Is it still a 1 image per batch limit by the way?

**Reply by IndigoDosSantos** [reply]
Awesome, we figured it out!

>Is it still a 1 image per batch limit by the way?

No, you're no longer limited to a single image, this issue got fixed. ✅
